Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -86,13 +86,45 @@ public class ApolloCodegen {
     config: ReferenceWrapped<ApolloCodegenConfiguration>,
     fileManager: FileManager = FileManager.default
   ) throws {
-    for graphQLObject in ir.schema.referencedTypes.objects {
+    for fragment in compilationResult.fragments {
```

Comment:
The referenced fields in the fragments and operations need to be collected before we generate the test mock objects, because we generate the `@Field` properties for each referenced field.